src/ripple/app/misc/SHAMapStoreImp.cpp

```diff
@@ -282,7 +286,8 @@ SHAMapStoreImp::run()
     while (1)
     {
         healthy_ = true;
-        validatedLedger_.reset();
+        validatedLedger.reset();
```

Comment:
I conisdered retrieving the validatedLedger through a lambda and declare and initialize it here. I wrote up what that would look like here: https://github.com/seelabs/rippled/commit/1f152385af53a5cb1526d97f8a64dcb7eaef1dae

I'm not recommending that because we need to move from a member variable into the return, and somebody is going to "optimize" that away and break the code. I also considered explicitly returning an rvalue ref from the lambda, but that gets ugly too because we sometimes need to return a null pointer, and we'd need a static for that. If it weren't for the `move` return danger I'd prefer that lambda.